extensions/cpsection/language/view.py

```diff
@@ -132,16 +134,20 @@ def _add_row(self, locale_code=None):
         language_palette = []
         key_list = self._language_dict.keys()
         for language_key in sorted(key_list):
+            locale = Locale(self._country_dict[language_key][0][0][:2])
```

Comment:
`[language_key][0][0][:2]` woah!

What does this mean?

Can we use one of the variables like locale_code?  Or can we give this a name?